website/static/overrides.css

```diff
@@ -389,8 +389,7 @@ em {
 }
 
 .usersSection .user {
-  height: 100px;
-  width: 200px;
+  height: 80px;
```

Comment:
what do you think about keeping width and changing height to: 
```suggestion
  max-height: 100px;
```
and align them vertically 
```css
  display: flex;
  align-items: center;
```

![image](https://user-images.githubusercontent.com/625469/72461203-c79d8000-37ce-11ea-81cf-cde638c63ead.png)